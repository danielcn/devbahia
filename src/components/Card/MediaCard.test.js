import React from 'react';
import { shallow } from 'enzyme';
import MediaCard from './MediaCard';
import Card from '@material-ui/core/Card';

describe('MediaCard Test', () => {
  it('Should render with snapshot', () => {
    const mediaCard = shallow(<MediaCard />);
    expect(mediaCard).toMatchSnapshot();
  });

  it('Shold have a child with with', () => {
    const styles = {
      card: {
        maxWidth: 345,
      },
      media: {
        height: 140,
      },
    };
    
    const mediaCard = shallow(<MediaCard />);
    expect(mediaCard.find('Card')).toHaveLength(1);
  });
});