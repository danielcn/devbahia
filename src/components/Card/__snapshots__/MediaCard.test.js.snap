// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaCard Test Should render with snapshot 1`] = `ShallowWrapper {}`;

exports[`MediaCard Test Suite Should have an image 1`] = `ShallowWrapper {}`;
